DZ_8/view.py

```diff
@@ -0,0 +1,66 @@
+import logger as log
+def show_main_menu():
+    while True:
+        data = input('''Выберете действие:
+        1 - Введите данные контакта
+        2 - Записать данные контакта в базу данных
+        3 - Отобразить содержимое БД
+        4 - Удалить запись в БД по id контакта
+        5 - Импортировать контаткы в базу из CSV
+        6 - Экспортировать контакты из базы в CSV
+        0 - Закончить работу
+        ''')
+        if data == '': break
+        if not data in ['1', '2', '3', '4', '5', '6', '7','8','9','10','0']:
```

Comment:
По вашей записи, вы проверяете вхождение `True` либо `False` в список, а не само значение `data`.
```suggestion
        if data not in ['1', '2', '3', '4', '5', '6', '7','8','9','10','0']:
```